recipes/xbout/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xbout" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 14a793b000882be6906345e623b5b5da7832cfdb0146cd97a9f53bb0f74df1ff
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<36, win]
```

Comment:
It looks like this package is pure Python. In this case, you can remove the `skip` here and add `noarch: python`.

Reply:
Also there should not be a comma

Reply:
Yes, looks like this messed up the builds. Thanks for catching @scopatz! BTW if having a comma here is malformed (it should have been `[py<36 or win]`, I think), shouldn't the linter catch that?